raiden/raiden_service.py

```diff
@@ -966,7 +966,7 @@ def _initialize_monitoring_services_queue(self, chain_state: ChainState):
         current_balance_proofs = views.detect_balance_proof_change(
             old_state=ChainState(
                 pseudo_random_generator=chain_state.pseudo_random_generator,
-                block_number=BlockNumber(1),
+                block_number=GENESIS_BLOCK_NUMBER,
```

Comment:
Looks like `GENESIS_BLOCK_NUMBER` is not typed properly. Can you fix this as you go. 🐭

Reply:
oops